src/test/java/sorald/processor/ProcessorDescriptionTest.java

```diff
@@ -0,0 +1,40 @@
+package sorald.processor;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.apache.commons.io.FilenameUtils;
+import org.junit.jupiter.api.Test;
+
+public class ProcessorDescriptionTest {
+    private static final Path PROCESSOR_PACKAGE = Paths.get("src/main/java/sorald/processor");
+
+    @Test
+    public void test_eachProcessorIsAccompaniedByDescription() {
+        List<File> processors =
+                Arrays.stream(PROCESSOR_PACKAGE.toFile().listFiles())
+                        .filter(file -> file.getName().endsWith(".java"))
+                        .collect(Collectors.toList());
```

Comment:
This is a little bit shaky, we can do better as Sorald knows all of the processors.

Add the following method to `Processors.java`:

```java
    /**
     * @return A list of all processors sorted by name.
     */
    public static List<Class<? extends SoraldAbstractProcessor<?>>> getAllProcessors() {
        return RULE_KEY_TO_PROCESSOR.values().stream()
                .sorted(Comparator.comparing(Class::getName))
                .collect(Collectors.toList());
    }
```

The sorting by name is just for consistency across invocations, makes testing more predictable.

Then rewrite this collection like so:

```suggestion
        List<File> processors = Processors.getAllProcessors().stream()
                .map(Class::getSimpleName)
                .map(procName -> PROCESSOR_PACKAGE.resolve(procName + ".java"))
                .map(Path::toFile);
```